setup.py

```diff
@@ -1,38 +1,8 @@
-import sys
-import setuptools
 import pathlib
+from setuptools import setup
+import setuptools_scm
 
-install_requires = ["pandas"]
-tests_require = ["pytest", "pytest-cov", "pytest-flake8", "asynctest"]
-dev_requires = install_requires + tests_require + ["documenteer[pipelines]"]
-scm_version_template = """# Generated by setuptools_scm
-__all__ = ["__version__"]
-
-__version__ = "{version}"
-"""
-tools_path = pathlib.PurePosixPath(setuptools.__path__[0])
-base_prefix = pathlib.PurePosixPath(sys.base_prefix)
-data_files_path = tools_path.relative_to(base_prefix).parents[1]
-
-setuptools.setup(
-    name="ts_m2",
-    description="This is a salobj 4 test package for the M2 component created "
-    "during a salobj 4 hackaton. ",
-    use_scm_version={
-        "write_to": "python/lsst/ts/m2/version.py",
-        "write_to_template": scm_version_template,
-    },
-    setup_requires=["setuptools_scm", "pytest-runner"],
-    install_requires=install_requires,
-    package_dir={"": "python"},
-    packages=setuptools.find_namespace_packages(where="python"),
-    package_data={"": ["*.rst", "*.yaml"]},
-    scripts=["bin/run_mtm2.py"],
-    tests_require=tests_require,
-    extras_require={"dev": dev_requires},
-    license="GPL",
-    project_urls={
-        "Bug Tracker": "https://jira.lsstcorp.org/secure/Dashboard.jspa",
-        "Source Code": "https://github.com/lsst-ts/ts_m2",
-    },
+setup(
+    version=setuptools_scm.get_version(),
+    scripts=[str(path) for path in pathlib.Path("bin").glob("*.py")],
```

Comment:
Please remove this line and the import for pathlib as well.

Reply:
Fixed in the commit: [dc78758](https://github.com/lsst-ts/ts_m2/pull/25/commits/dc78758f743d2867d90bbb8500248fe6606a8870). Thanks!